lib/isgeocsv.js

```diff
@@ -0,0 +1,20 @@
+var geocsvinfo = require('geocsv-info');
```

Comment:
> a separate CSVdetect module

The test here is specifically targeting the existence of geocsv headers. If the first line can be parsed into a valid CSV header with some geocsv fields, the test comes back positive.

I'm not sure what the proper convention would be here. I feel like we are taking up a lot of tests in this repo that are just checking the validity of our `isgeocsv` function. It would be nice to modularize these tests as well as some more robust tests against all of the [mapnik test csv's](https://github.com/mapnik/test-data/tree/master/csv) (see below).